ReactiveTask/Task.swift

```diff
@@ -545,16 +545,18 @@ public func launchTask(taskDescription: Task, standardInput: SignalProducer<NSDa
 						dispatch_group_leave(group)
 					}
 
-					task.launch()
+					rawTask.launch()
 					close(stdoutPipe.writeFD)
 					close(stderrPipe.writeFD)
 
+					observer.sendNext(.Launch(task))
```

Comment:
This looks like it might be a little tricky. Since I forgot to do this in the original PR, I can take over this PR if you'd like. But I